src/libs/ReportUtils.js

```diff
@@ -2772,6 +2772,16 @@ function canAccessReport(report, policies, betas, allReportActions) {
 
     return true;
 }
+/**
+ *
+ * @param {Object} report
+ * @param {String} currentReportId
+ * @returns {Boolean}
+ */
+function shouldHideReport(report, currentReportId) {
```

Comment:
This function is a bit vague. I kinda think we should merge this and `isChildReportHasComment` into a single `shouldHideReport` helper function.